.travis.yml

```diff
@@ -4,12 +4,12 @@ matrix:
   include:
     - python: 2.7
       env: TOXENV=py27-sdist
-    - python: 3.6
-      env: TOXENV=py36-sdist
     - python: 3.7
       env: TOXENV=py37-sdist
     - python: 3.8
-      env: TOXENV=py38-sdist,formatting,docs
+      env: TOXENV=py38-sdist
+    - python: 3.11
```

Comment:
Trivial: If we support Python 3.9 and 3.10 as well, should we include those versions here?
I'd be fine with dropping Python 2.7 and even 3.7 if we want to keep the number of supported Python versions the same.

Reply:
Yeah that's probably a better idea. I'd like to keep 2.7 around I think for now at least, just cause we still actually support it in the tests and what not